src/style.css

```diff
@@ -1 +1,72 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+  background-color: #00acdc;
```

Comment:
Move all repeatable colors to the variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)